apps/preview/nuxt/pages/showcases/Link/NuxtLink.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="space-x-4">
-    <SfLink :tag="NuxtLink" :to="{ path: '#', query: { docs: true } }"> NuxtLink </SfLink>
+    <SfLink :tag="NuxtLink" :to="{ path: '#' }"> NuxtLink </SfLink>
```

Comment:
I think, now it could be just 
```suggestion
    <SfLink :tag="NuxtLink" to="#"> NuxtLink </SfLink>
```